CONTRIBUTING.md

```diff
@@ -151,29 +151,11 @@ Leave that running. And in a new window, you can run the tests:
 yarn test
 ```
 
-If everything is set up correctly, you should see the following happen
-automatically:
-1. Chrome browser starts.
-1. The URL for one of the sample CodePen projects loads.
-1. Layout is changed for better viewing.
-1. External javascript settings are changed to use your local copy of bundle.js.
-1. The CodePen is re-run to get the new bundle.js.
-1. The "Run Tests" button is clicked and the script waits for test to complete.
-1. The "Tests" button is clicked to see the test output.
-1. A screenshot of the results is saved.
-1. Move on to the next CodePen by returning to step 1.
-
 You will find screenshots of all the test results in the `test/screenshots`
 directory (or the directory you configured in your `test/setup.js` file).
 
 #### Testing tips
 
-**Turn your volume down**
```

Comment:
Why did this get removed? Is there no sound when in headless mode?

Reply:
No, it is not because of headless mode. I turn off sounds in settings for Chrome and Firefox.

Reply:
I'm not sure what you mean by this? Do you personally turn off sounds on your preferences? Or is there some code here that turns it off so no one will ever hear the sounds during testing?

Reply:
some code

Reply:
Cool. Can you point me to that code? I couldn't find it.

Reply:
Nevermind, I found it. It was hidden because of github not showing large diffs.